src/server/main.py

```diff
@@ -1,46 +1,66 @@
+"""Module enables socket creation and listen to client"""
+import socket
 from utils.serialise import Serialiser
 from utils.decrept import Decryption
 from utils.file_handler import FileHandler
-import socket
+
+
+def convert_file(input_data, format_input):
+    """
+    Helper function that creates a txt file then convertes it to the desired format
+    """
+    txt = FileHandler.create_txt(input_str=input_data)
+    output_file = FileHandler.convert_file(file=txt, format_type=format_input)
+    return output_file
 
 
 def receive_from_client(client_data):
+    """
+    Main function that process data sent from the client
+    """
     deserialiser = Serialiser()
     decrypter = Decryption()
     deserialised = deserialiser.deserialise(client_data)
     if deserialised["ENCRYPTION"]:
         decrypted_string = decrypter.decrypt_message(deserialised["INPUT_STRING"])
         if deserialised["PRINT_OUTPUT"]:
-            output_str = "The following message received from client:\n {}".format(decrypted_string)
-            print(output_str)
-            return output_str
+            requested_output = f"The following message received from client:\n {decrypted_string}"
+            print(requested_output)
         elif deserialised["TEXT_FILE_OUTPUT"]:
-            txt = FileHandler.create_txt(input_str=decrypted_string)
-            output_file = FileHandler.convert_file(file=txt, format_type=deserialised["OUTPUT_TYPE"])
-            return output_file
+            requested_output = convert_file(input_data=decrypted_string,
+                                            format_input=deserialised["OUTPUT_TYPE"])
+        else:
+            print("Request not supported")
     else:
         message = deserialised["INPUT_STRING"]
         if deserialised["PRINT_OUTPUT"]:
-            output_str = "The following message received from client:\n {}".format(message)
-            print(output_str)
-            return output_str
+            requested_output = f"The following message received from client:\n {message}"
+            print(requested_output)
         elif deserialised["TEXT_FILE_OUTPUT"]:
-            txt = FileHandler.create_txt(input_str=message)
-            output_file = FileHandler.convert_file(file=txt, format_type=deserialised["OUTPUT_TYPE"])
-            return output_file
-    return
-
+            requested_output = convert_file(input_data=message,
+                                            format_input=deserialised["OUTPUT_TYPE"])
+        else:
+            print("Request not supported")
+    return requested_output
 
 def create_listening_socket():
```

Comment:
It might be a better idea to put the variables "host" and "port" in a config file.